cairo/tests/ethereum/cancun/vm/test_runtime.py

```diff
@@ -7,13 +7,15 @@
 from cairo_addons.testing.errors import cairo_error
 from tests.utils.solidity import get_contract
 
+pytestmark = pytest.mark.python_vm
```

Comment:
What is missing to pass it to rust?
Could we make a follow up issue

Same where relevant

Reply:
`test_runtime.cairo` is full of hints in tests for now, this will be covered by #774